test/e2e/webhook_test.go

```diff
@@ -424,3 +426,1021 @@ var _ = Describe("webhook tests for MC taints", Ordered, func() {
+									{
+										Operator: placementv1alpha1.JSONPatchOverrideOpRemove,
+										Path:     "/meta/labels/test-key",
+										Value:    apiextensionsv1.JSON{Raw: []byte(`"test-value"`)},
+									},
+									{
+										Operator: placementv1alpha1.JSONPatchOverrideOpReplace,
+										Path:     "/kind",
+										Value:    apiextensionsv1.JSON{Raw: []byte(`"new-kind"`)},
+									},
+									{
+										Operator: placementv1alpha1.JSONPatchOverrideOpReplace,
+										Path:     "////",
+										Value:    apiextensionsv1.JSON{Raw: []byte(`"new-kind"`)},
+									},
+								},
+							},
+						},
+					},
+				},
+			}
+			By(fmt.Sprintf("expecting denial of CREATE override %s", cro.Name))
+			err := hubClient.Create(ctx, cro)
+			var statusErr *k8sErrors.StatusError
+			Expect(errors.As(err, &statusErr)).To(BeTrue(), fmt.Sprintf("Create CRO call produced error %s. Error type wanted is %s.", reflect.TypeOf(err), reflect.TypeOf(&k8sErrors.StatusError{})))
+			Expect(statusErr.Status().Message).Should(MatchRegexp("remove operation cannot have value"))
+			Expect(statusErr.Status().Message).Should(MatchRegexp("cannot override typeMeta fields"))
+			Expect(statusErr.Status().Message).Should(MatchRegexp("path cannot contain empty string"))
+			return nil
+		}, testutils.PollTimeout, testutils.PollInterval).Should(Succeed())
+	})
+
+	It("should deny create CRO for a resource already selected by another CRO", func() {
+		Eventually(func(g Gomega) error {
+			By("Create the 1st ClusterResourceOverride")
+			cro := &placementv1alpha1.ClusterResourceOverride{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: croName,
+				},
+				Spec: placementv1alpha1.ClusterResourceOverrideSpec{
+					ClusterResourceSelectors: []placementv1beta1.ClusterResourceSelector{
+						selector,
+					},
+					Policy: policy,
+				},
+			}
+			By(fmt.Sprintf("expecting denial of CREATE override %s", cro.Name))
+			err := hubClient.Create(ctx, cro)
+			Expect(err).NotTo(HaveOccurred(), "Failed to create CRO %s", cro.Name)
+
+			By("Try to create the 2nd ClusterResourceOverride")
+			cro2 := &placementv1alpha1.ClusterResourceOverride{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: fmt.Sprintf("test-cro-%d", GinkgoParallelProcess()),
+				},
+				Spec: placementv1alpha1.ClusterResourceOverrideSpec{
+					ClusterResourceSelectors: []placementv1beta1.ClusterResourceSelector{
+						selector,
+					},
+					Policy: policy,
+				},
+			}
+			err = hubClient.Create(ctx, cro2)
+			var statusErr *k8sErrors.StatusError
+			Expect(errors.As(err, &statusErr)).To(BeTrue(), fmt.Sprintf("Create CRO call produced error %s. Error type wanted is %s.", reflect.TypeOf(err), reflect.TypeOf(&k8sErrors.StatusError{})))
+			Expect(statusErr.Status().Message).Should(MatchRegexp(fmt.Sprintf("invalid resource selector %+v: the resource has been selected by both %v and %v, which is not supported", selector, cro2.Name, cro.Name)))
+			cleanupClusterResourceOverride(croName)
```

Comment:
can you use Before and After func to setup the tests? to improve the readability.